src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -91,6 +88,19 @@ MulticopterAttitudeControl::MulticopterAttitudeControl() :
 	}
 
 	parameters_updated();
+
+	_gyro_count = math::constrain(orb_group_count(ORB_ID(sensor_gyro)), 1, MAX_GYRO_COUNT);
+}
+
+MulticopterAttitudeControl::~MulticopterAttitudeControl()
+{
+	perf_free(_loop_perf);
+}
+
+bool
+MulticopterAttitudeControl::init()
+{
+	return selected_gyro_update();
```

Comment:
Do you select the gyro on module start "only"?

Reply:
Why was I already outdated? 🤷‍♂

Reply:
I think it was commenting on something in the 1st commit (mc_att_control changes) that changed again in the 2nd (moving this out of mc_att_control entirely)?

Reply:
> Do you select the gyro on module start "only"?

It also updates if `sensor_selection` (from sensors voting) changes as well.